### ISSUE TITLE
Lambda Value Range

### ISSUE THREAD

**Issue by ZSBSB** [huzi96/Coarse2Fine-ImaComp#10]
Hi, I just started learning about image compression. Does the lambda in the loss function have a suggested range of values?

**Comment by huzi96** [comment]
Hi, you may want to check https://arxiv.org/pdf/2002.03711.pdf section 6.5 for the lambdas I used. You may also want to check the repo https://github.com/huzi96/Coarse2Fine-PyTorch for a training code.

**Comment by ZSBSB** [comment]
![image](https://user-images.githubusercontent.com/39560063/170047909-09844a43-10b2-481f-90a1-0b132d47f2b1.png)
okay, thank you
Where are the JPEG and BPG compression models in Figure 5 in your paper found?

**Comment by ZSBSB** [comment]
OK, thank you
